easybuild/toolchains/fft/fftw.py

```diff
@@ -53,8 +53,12 @@ def _set_fftw_variables(self):
         fftw_libs = ["fftw%s" % suffix]
         if self.options['usempi']:
             fftw_libs.insert(0, "fftw%s_mpi" % suffix)
+        fftw_libs_mt = ["fftw%s" % suffix]
+        if self.options['openmp', False]:
```

Comment:
@boegel What am I doing wrong here? The tests are still failing...should the default be none?